torch_ttnn/passes/to_tt_pass.py

```diff
@@ -368,6 +368,17 @@ def ReplaceMoreTtManually(gm: torch.fx.GraphModule) -> torch.fx.GraphModule:
                     new_node,
                     delete_user_cb=lambda node: node != new_node,
                 )
+            if node.target == torch.ops.aten.transpose.int:
```

Comment:
Is there an in_place version of this operation?

Reply:
Not that I am aware of.